css/template-2.css

```diff
@@ -101,3 +135,32 @@
 .isi p a {
   color: #eaba3a;
 }
+
+.spec {
+  display: inline-block;
+  color: #515151;
+}
+
+.spec p {
+  margin-bottom: 0.2em;
+}
+
+.tab-spec .active {
+  border-bottom: #eaba3a 2px solid;
+}
+
+.tab-spec li a {
```

Comment:
Codacy found an issue: [Expected selector ".tab-spec li a" to come before selector ".list-aside li a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)